packages/p2p/src/components/modal-manager/modal-manager-context-provider.tsx

```diff
@@ -1,15 +1,54 @@
 import React from 'react';
-import { useStores } from 'Stores';
+import { useStores } from 'Stores/index';
 import { ModalManagerContext } from './modal-manager-context';
 import { isDesktop } from '@deriv/shared';
+import type { TModalKeys, TModalProps } from 'Constants/modals';
 
-const ModalManagerContextProvider = props => {
-    const [active_modal, setActiveModal] = React.useState({});
-    const [previous_modal, setPreviousModal] = React.useState({});
+/**
```

Comment:
`suggestion` maybe it would be better to separate out all the typings here into its own type file? since theres quite a lot instantiated here